pkg/ui/static/js/bucket.js

```diff
@@ -7,6 +7,9 @@ function draw() {
     if (thanos.refreshedAt == "0001-01-01T00:00:00Z") {
         thanos.err = "Synchronizing blocks from remote storage";
     }
+    else if (thanos.blocks.length === 0) {
```

Comment:
We should check here also if `thanos.err` is not null because otherwise, we will hide the "real" error here because our backend gives us an empty array when an error happens.

Reply:
@GiedriusS You mean if `thanos.err` *is* null? e.g. `!thanos.err && thanos.blocks.length == 0`

Reply:
Yes, that's what I mean :+1: